Sources/Core/Microsoft.StreamProcessing/StreamableAPI/GroupAggregateTemplate.tt

```diff
@@ -17,8 +17,19 @@ namespace Microsoft.StreamProcessing
     public static partial class Streamable
     {
         /// <summary>
-        /// Groups input events by a key selector and applies an aggregate to "snapshot windows" (SI terminology) on each group.
+        /// Groups input events by a key selector and applies an aggregate on each group.
         /// </summary>
+        /// <typeparam name="TOuterKey">The type of the grouping key of the stream coming into the aggregation operation.</typeparam>
+        /// <typeparam name="TInput">The type of the payload in the input stream.</typeparam>
+        /// <typeparam name="TInnerKey">The result type of the expression used to group data in the aggregation.</typeparam>
+        /// <typeparam name="TState1">The type of the state object maintained by the aggregate operation.</typeparam>
+        /// <typeparam name="TOutput1">The type of the results generated by the aggregate operation.</typeparam>
+        /// <typeparam name="TOutput">The type of the payloads of the resulting stream.</typeparam>
```

Comment:
While you're here, maybe TOutput and TOutput1 should be renamed to be more meaningful?

Reply:
I'll take care of this in the next checkin. Thanks!